src/main/scala/io/iohk/ethereum/domain/SignedTransaction.scala

```diff
@@ -94,6 +96,25 @@ object SignedTransaction {
       sender <- SignedTransaction.getSender(tx, signature, recoveredPointSign)
     } yield SignedTransaction(tx, signature, sender)
   }
+
+  def sign(tx: Transaction, keyPair: AsymmetricCipherKeyPair): SignedTransaction = {
+    val bytes = crypto.kec256(
```

Comment:
The signing method has changed slightly. When receiving transactions we need to deal with both the current and the historic one.

But we will only be signing new transactions, so only the current method will be used.

It would still be nice to avoid the code repetition, but let's close one eye for now 😉